*** /var/local/brisskit/drupal/site/civicrm/sites/all/modules/civicrm/CRM/Core/DAO.php	2015-03-06 03:16:37.000000000 +0000
--- ./DAO.php	2016-02-05 00:12:38.708652523 +0000
***************
*** 364,369 ****
--- 364,372 ----
    function initialize() {
      $this->_connect();
      $this->query("SET NAMES utf8");
+ 
+     ### The included call file runs method BK_Hooks::set_component_name() which ensures our view returns what we expect.
+     include('brisskit_DAO.php');
    }
  
    /**
